### PR TITLE
Delete unused file, add easylog into unitest main, add local coverage tools

### PR DESCRIPTION
**Pull request by JimmyShi22**
(1) Delete unused file
(2) Add easylog into unitest main file
(3) Add local coverage tools
(4) Add trie and MenTrie unitest
(5) Translate some original easylogging++ unitest to lab-bcos

### CODE REVIEW

File: test/tools/fisco-bcos-test/main.cpp [hunk 1]
```diff
@@ -17,9 +17,11 @@
  * @brief: empty test framework of lab-bcos
  *
  * @file main.cpp
- * @author: yujiechen
+ * @author: yujiechen, jimmyshi
```
Comment (wheatli) [inline test/tools/fisco-bcos-test/main.cpp]: pls add a new line to indicates your modification

File: tools/generate_coverage.sh [hunk 2]
```diff
@@ -0,0 +1,176 @@
+ # "Copyright [2018] <fisco-bcos>"
+ # @ function : one-click install shell script(appliable for centos and ubuntu)
+ # @ Require  : yum or apt, git are ready
+ # @ attention: if dependecies are downloaded failed,
```
Comment (wheatli) [inline tools/generate_coverage.sh]: pls make sure the message you copy to here

File: test/tools/fisco-bcos-test/main.cpp (hunk 1 above)
Comment (JimmyShi22) [reply test/tools/fisco-bcos-test/main.cpp]: Add a more words to describe this cpp

File: tools/generate_coverage.sh (hunk 2 above)
Comment (JimmyShi22) [reply tools/generate_coverage.sh]: done

**Comment by JimmyShi22** [comment]
Sorry for wrong dismissed the review of wheatli....

**Pull request merged**